### PR TITLE
Revert aws marketplace changes

### PR DESCRIPTION
**Pull request by brucehoff**
This reverts all the changes we made to support AWS marketplace.

### CODE REVIEW

File: src/main/java/synapseawsconsolelogin/Auth.java [hunk 1]
```diff
@@ -97,9 +92,6 @@
 	static final String AWS_REGION_PARAMETER = "AWS_REGION";
 	static final String SYNAPSE_OAUTH_CLIENT_ID_PARAMETER = "SYNAPSE_OAUTH_CLIENT_ID";
 	static final String SYNAPSE_OAUTH_CLIENT_SECRET_PARAMETER = "SYNAPSE_OAUTH_CLIENT_SECRET";
-	static final String MARKETPLACE_PRODUCT_CODE_SC_PARAMETER = "MARKETPLACE_PRODUCT_CODE_SC";
-	static final String MARKETPLACE_ID_DYNAMO_TABLE_NAME_PARAMETER = "MARKETPLACE_ID_DYNAMO_TABLE_NAME";
```
Comment (zaro0508) [inline src/main/java/synapseawsconsolelogin/Auth.java:101]: these are params in the AWS parameter store.  should we remove these parameters from there as well?

File: src/main/webapp/WEB-INF/web.xml [hunk 2]
```diff
@@ -13,8 +13,6 @@
 		<url-pattern>/synapse</url-pattern>
 		<url-pattern>/</url-pattern>
 		<url-pattern>/health</url-pattern>
-		<url-pattern>/about</url-pattern>
```
Comment (zaro0508) [inline src/main/webapp/WEB-INF/web.xml:16]: why is this removed?  i think we want to keep it.

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/main/webapp/WEB-INF/web.xml (hunk 2 above)
Comment (brucehoff) [reply src/main/webapp/WEB-INF/web.xml:16]: Yes, you are right. I've updated the PR.

File: src/main/java/synapseawsconsolelogin/Auth.java (hunk 1 above)
Comment (brucehoff) [reply src/main/java/synapseawsconsolelogin/Auth.java:101]: Yes, you are right.  We first have to revert changes in other repo's, including synapse-login-aws-infra and organizations-infra.  I will create PRs for those changes.

File: src/main/java/synapseawsconsolelogin/Auth.java [hunk 3]
```diff
@@ -187,13 +160,8 @@ private void initialize() {
 		return getCommaSeparatedPropertyAsList(REDIRECT_URIS_PROPERTY_NAME, defaultRedirectURI);
 	}
 
-	/**
-	 * @return the URL to redirect the client to for OAuth login.
-	 */
-	public String getAuthorizeUrl() throws UnsupportedEncodingException {
-		// userid claim is used by this application so it always must be included in the list of claims requested from Synapse
-		Set<String> allClaims = new TreeSet<String>(Collections.singleton(USER_ID_CLAIM_NAME));
```
Comment (zaro0508) [inline src/main/java/synapseawsconsolelogin/Auth.java:195]: should we keep the `throws UnsupportedEncodingException`?

should we keep the `Collection.singleton`?

**Review by zaro0508 — CHANGES_REQUESTED** [review]

File: src/main/java/synapseawsconsolelogin/Auth.java (hunk 3 above)
Comment (brucehoff) [reply src/main/java/synapseawsconsolelogin/Auth.java:195]: No:  (1) The removed line 195 added a claim not requested by the configuration but needed by the Marketplace addition.  With the removal of the Marketplace code this is no longer needed.  (2) With the removal of the code shown in red, the function `getAuthorizedUrl` no longer throws UnsupportedEncodingException so it's removed from the method signature.

More generally, this PR is an unwinding of the code added for Marketplace. With few exceptions (i.e., some code clean up) the PR is simply a removal of the added code.

**Review by zaro0508 — APPROVED** [review]
